packages/gcm/lib/src/gcm.dart

```diff
@@ -0,0 +1,22 @@
+// Copyright 2015, the Flutter authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:flutter/services.dart';
+import 'package:sky_services/gcm/gcm.mojom.dart';
```

Comment:
Ah, right.

Yeah we definitely don't want anything to end up in the engine repo for this eventually. But again, as you say, this is fine for initial discovery of the problem space. Might be worth putting comments around this saying this is all temporary and that eventually third-party services will have their own repo(s), just in case someone looks at the code in the meantime...